### DISCUSSION TITLE
Dec 2023 Review - PR1: Graphing Quadratic Functions

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/precalculus#40] [Show and tell]
Section: https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html

Author: @tdegeorge 

Primary Reviewer: @AbbyANoble (feedback by Dec 11 morning)

**Comment by AbbyANoble** [comment]
[Activity 1.1.1](https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html#activity-1) 

There is a table, but they students aren't asked to fill it out. Maybe rewrite to something like:

(a) Make a table for values of f(x) corresponding to the given x-values. What is happening to the y-values as the x-values increase? Do you notice any other patterns of the y-values of the table?

(I'd also include one more x value off the graph, like -1 so they could make the connection that the graph keeps going beyond what we see here.)

For part (b), I think it might be good to consider maximum and minimum values separately to get them to think about why quadratics have one but not the other. For example, 

(b) At which point (x,y) does f(x) have a minimum value? That is, is there a point on the graph that is lower than all other points?

- There is no minimum value of this function.
- It seems to occur near (-1/5,10)
- It seems to occur near (2,-4)
-  [some other answer?]

(c) At which point (x,y) does f(x) have a maximum value? That is, is there a point on the graph that is higher than all other points?

- There is no maximum value of this function.
- It seems to occur near (-1/5,10)
- It seems to occur near (2,-4)
- It seems to occur near (4 1/5,10)
- [some other answer?]


In (c), rather than appealing to a "faster" way, I would say "Do you see a connection between the function as it is written and the minimum value?" Or skip (c) all together, give the definition, then go into activity 1.1.5.

**Reply by siwelwerd** [reply]
For 1.1.1, several of the parts ask the students multiple questions.  I'd suggest making these more **Specific** asks of the students.  I think 1.1.5 does a much better job of being **Specific**.  I like Abby's suggestions, they are one (but not the only) way of doing this.

**Comment by AbbyANoble** [comment]
For definitions, include the `<term> </term>` tag around the word(s) being defined so they stand out.

**Reply by siwelwerd** [reply]
Often, we also want to include an index tag as well, which will make an index entry pointing to that point.  They syntax for that is 
`<term>vertex form</term><idx><h>quadratic function</h><h>vertex form</h></idx>` .  Note that everything in the `<idx>` tag does not show up here, but in the index.  The `<h>`s tells the index to put 'vertex form' under 'quadratic function'.  

Cheatsheet documentation on index entries: https://rellek.net/ptx/cheatsheet/s-small.html#ss-idx

**Reply by AbbyANoble** [reply]
Thanks! I was going to look that up and add it in!

**Reply by tdegeorge** [reply]
Ok, I am going to attempt to address this too.....please check it though when I am done! :)

**Comment by AbbyANoble** [comment]
[Activity 1.1.5](https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html#activity-2) 

You could make part (a) MC, especially playing around with the negatives to make sure they write it correctly.

A. vertex (2,-4), axis x=2
B. vertex (-2,4), axis x=-2
C. vertex (-2,-4), axis x=-2
D. vertex (2,4), axis x=2

**Comment by AbbyANoble** [comment]
[Activity 1.1.7](https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html#activity-3) 

in (a), B and D are the same answer

I'd also say "could be" instead of "matches" since they haven't learned enough to say it's that one for sure.

**Comment by AbbyANoble** [comment]
[Activity 1.1.11](https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html#activity-5) 

I think this could be expanded to get them thinking about zeros as well. 

A function f(x) has a maximum value at 7 and its axis of symmetry at x=-2. 

(a) Sketch a graph of a function that meets the criteria for f(x). 
(b) Was your graph the only possible answer? Try to sketch another graph that meets this criteria.

Then some sort of remark that some other points may be helpful in sketching the graph, and that finding the zeros will help us graph it more accurately.

Activity on finding zeros. (And we need to include factoring/solving quadratics in the readiness assurance.) Then leads to the Check It problem, which could be included at the end as a fluency builder.

**Reply by tdegeorge** [reply]
So, I added the remark, changed the activity, and then added another activity about zeros....Not sure it is what you had in mind, but once I submit a pull request, we can look at it again! Also, the objective does not have zeros in it - do we want to add that?

**Comment by siwelwerd** [comment]
The intro text before Activity 1.1.1 is in a `<p>`, but not within anything else.  As a result, it will not show up in the [generated slides](https://teambasedinquirylearning.github.io/precalculus/2024/pdf/TBIL-Precalculus-2024-slides.pdf).  Typically we want to have an `<observation>` that includes this kind of text, which will show up in slides. See #29

**Reply by tdegeorge** [reply]
I am not sure I did this right.....Please check this when I submit my pull request! :)

**Reply by siwelwerd** [reply]
Will do!  You can also leave comments when you make the pull request on any particular thing you want reviewed.  This is extra helpful if you go to your PR page on github, and click on "Files changed" across the top.  Then you can scroll down to the particular line of code you changed, and if you hover a "+" will appear and you can drop a comment right at the particular place.

**Comment by AbbyANoble** [comment]
[Activity 1.1.1(b)](https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html#task-2) 

Typo - needs to be "its" instead of "it's"

**Comment by tdegeorge** [comment]
@siwelwerd is there a way to refer to a specific definition? Like, rather than saying "apply the definition," I want to say something like "apply definition 1.2.1"

**Reply by siwelwerd** [reply]
Yes, use an xref pointing to the xml:id of the thing you want to reference.  So for example, `<xref ref="vertex-form-of-a-quadratic-function">` will produce [Definition 1.1.4]( https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html#vertex-form-of-a-quadratic-function) , since the xml:id specified for Definition 1.1.4 was "vertex-form-of-a-quadratic-function".

**Comment by siwelwerd** [comment]
@tdegeorge's revisions are now live in the book! https://teambasedinquirylearning.github.io/precalculus/2024/PR1.html
